src/libraries/Microsoft.Extensions.Logging/pkg/Microsoft.Extensions.Logging.pkgproj

```diff
@@ -4,6 +4,7 @@
     <ProjectReference Include="..\src\Microsoft.Extensions.Logging.csproj">
       <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
+    <ProjectReference Include="..\gen\Microsoft.Extensions.Logging.Generators.csproj" />
```

Comment:
I thought the Generator was going to come in a separate nuget package initially.

If we put it in this package - how is an ASP.NET application going to use the source generator?

Reply:
- [ ] for ASP.NET Core we need to make sure they get the generator through the transport package.
But might change after preview 4

cc: @ericstj

Reply:
They can reference this package. We don’t need a different one.   Conflict resolution will drop unnecessary assets that are present in framework.

Once we close on inbox generators the reference will not be needed.